docker-compose.yml

```diff
@@ -0,0 +1,39 @@
+version: '3.4'
+
+services:
+    php:
+        build:
+            context: .
+            target: server-for-symfony-flex-php
+            args:
+                - FPM_PORT=${FPM_PORT:-9000}
+        env_file:
+            - .env.local
```

Comment:
I'd prefer having a proper .env.docker - this way the setup can be prepared to work from the start. Would you mind adding one?

Reply:
Yes and no :) That configuration file should not be committed to the repo because it will potentially contain sensitive information. Currently the idea is to copy the `.env` file to `.env.local`, but I have no problem with changing the name to `.env.docker`, however I'm not sure Symfony will pick up on that file when not using Docker, as is the case with `.env.local` (so it works like the "normal" situation).

So maybe add something like `.env.docker.dist`?

Reply:
@moay Just waiting for an answer on this one.

Reply:
You're right. I was thinking of adding some preset that would work oob where you just have to change stuff like adding your own recipes repo. But of course this would go to the repo, so maybe not the best solution.